public/pages/CreateMonitor/components/ClusterMetricsMonitor/utils/clusterMetricsMonitorConstants.js

```diff
@@ -104,6 +104,25 @@ export const API_TYPES = {
       source: 'ctx.results[0].nodes.NODE_ID.jvm.mem.heap_used_percent > 60',
     },
   },
+  CAT_INDICES: {
+    type: 'CAT_INDICES',
+    documentation: 'https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-indices/',
+    exampleText: 'index1,index2...',
+    label: 'List indices',
+    paths: {
+      withPathParams: '_cat/indices',
+      withoutPathParams: '_cat/indices',
```

Comment:
Doubt: Both should be same?

Reply:
Yes, that's fine for now. Some of the other API use that same framework. I'll take an action item to implement regex to accomplish this; but for now, this framework supports easy extensibility.